aspnetcore/blazor/hosting-models.md

```diff
@@ -103,6 +103,187 @@ The graph is rerendered, and a UI *diff* (difference) is calculated. This diff i
 
 A component is disposed after the user navigates away from it on the client. While a user is interacting with a component, the component's state (services, resources) must be held in the server's memory. Because the state of many components might be maintained by the server concurrently, memory exhaustion is a concern that must be addressed. For guidance on how to author a Blazor Server app to ensure the best use of server memory, see <xref:security/blazor/server>.
 
+### Integrate Razor components into Razor Pages and MVC apps
+
+#### Use components in pages and views
+
+An existing Razor Pages or MVC app can integrate Razor components into pages and views:
+
+1. In the app's layout file (*_Layout.cshtml*):
+
+   * Add the following `<base>` tag to the `<head>` elements:
+
+     ```html
+     <base href="~/" />
```

Comment:
Your app might not be hosted at the root, right? I believe this needs to match whatever the base address of your app is.